firstvoices/backend/models/song.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+from backend.permissions import predicates
+from .base import AudienceMixin, BaseControlledSiteContentModel
+from .media import RelatedMediaMixin
+from .mixins import AuthouredMixin, dynamic_translated_field_mixin_factory
+
+
+class Song(
+    AuthouredMixin,
+    AudienceMixin,
+    RelatedMediaMixin,
+    dynamic_translated_field_mixin_factory("Song",
+                                           "title",
+                                           unique=False,
+                                           blank=False,
+                                           nullable=False
+                                           ),
+    dynamic_translated_field_mixin_factory("Song", "introduction"),
+    dynamic_translated_field_mixin_factory("Song", "lyrics"),
+    BaseControlledSiteContentModel,
+):
+    """
+    Representing a song associated with a site, including unique title, lyrics, introduction, and media links
+
+    Notes for data migration:
+    authours from fvbook:author (which should be dereferenced)
```

Comment:
Ohhhh sorry, I probably forgot to specify the plan for mapping author data. Instead of keeping the author field, we want to migrate that data into the acknowledgements field just as string values. Full info on the field mapping wiki page: https://firstvoices.atlassian.net/wiki/spaces/FIR/pages/79691807/V2+Data+Migration+Field+Mapping#FVSong*